src/types-and-traits.rst

```diff
@@ -2749,3 +2758,67 @@ its lifetime elided form is
 
    fn f <T: ToCStr>(&mut self, args: &[T]) -> &mut Command;
 
+.. _fls_XTBOhK2Yk4lA:
+
+Trait Object Lifetime Elision
```

Comment:
Perhaps we want to define "trait object lifetime elision" the same way it was defined for function lifetime elision. An alternative might be to leverage the term "lifetime elision", for example:

Function lifetime elision is a form of lifetime elision that applies to functions ...